lib/manager/composer/artifacts.ts

```diff
@@ -96,28 +96,28 @@ export async function updateArtifacts(
       await fs.outputFile(localAuthFileName, JSON.stringify(authJson));
     }
     const startTime = process.hrtime();
-    let cmd: string;
+    const cmd = 'composer';
+    const execOptions: ExecOptions = {
+      cwd,
+      env,
+      inheritEnvVars: ['COMPOSER_CACHE_DIR'],
```

Comment:
@rarkins I think I would need to make `inheritEnv` instead, which can be Object like `env`, but filled with fallback values for missing `process.env` keys.

Reply:
In this case we'll use it instead of "raw" `env` which still can be used to "force" some variables

Reply:
Or maybe it's okay to have `process.env` vars priority over every `env` value

Reply:
Does it make sense if we call it something like `extraEnvVars` ?